lang/en.json

```diff
@@ -9,6 +9,8 @@
   "Remaining Time": "Remaining Time",
   "Stream Type": "Stream Type",
   "LIVE": "LIVE",
+  "Seek to LIVE": "Seek to LIVE",
```

Comment:
we should run `npm run docs:lang` to update the translations-needed doc.

Reply:
Will do that now, just so we remember, but I am not sold on what it should actually say yet.

Reply:
we should remember to do this right before merging